thirdparty/github.com/go-yaml/yaml.v3/yaml.go

```diff
@@ -1,698 +1,808 @@
 //
 // Copyright (c) 2011-2019 Canonical Ltd
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+// Copyright (c) 2006-2010 Kirill Simonov
```

Comment:
curious where this diff came from

Reply:
May have been a copy-paste error; you're right that there shouldn't be a copyright diff in this commit. Will update